src/main/java/org/broadinstitute/dsm/db/ParticipantEvent.java

```diff
@@ -133,5 +134,23 @@ public static void skipParticipantEvent(@NonNull String ddpParticipantId, @NonNu
         }
         return skippedEvents;
     }
+
+    public static Collection<String> getParticipantEvent(Connection conn, @NonNull String ddpParticipantId, @NonNull String instanceId) {
+        ArrayList<String> skippedEvents = new ArrayList();
+        try (PreparedStatement stmt = conn.prepareStatement(TransactionWrapper.getSqlFromConfig(ApplicationConfigConstants.GET_PARTICIPANT_EVENT))) {
```

Comment:
can you move sql for `GET_PARTICIPANT_EVENT` from the application.conf to the class, please